deploy/Makefile

```diff
@@ -3,17 +3,17 @@ SHELL := /bin/bash
 bots-namespace:
 	kubectl apply -f cluster/bots_namespace.yaml
 
-app-config:
-	kubectl apply -f config/app_config.yaml
+ti-sync-bot-config:
+	kubectl apply -f config/ti_sync_bot_config.yaml -n ti-community-bots
```

Comment:
Please put namespace in yaml and not in the command.

Reply:
We don't seem to need to maintain these configurations in this repository, do we?

Reply:
Prod configuration can be managed uniformly in configs repo, but as a project, it should also include complete files including deployment files, similar to the configuration in tichi's prow-dev.

Reply:
Maybe we could create a dev folder, as I think it might be complicated if each bot maintains these deployment files separately.

Reply:
Since we may have multiple bots, it would be very complicated if we need to clone and deploy them separately.

Reply:
I thought for a while, you are right.